.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "data/sigma"]
```

Comment:
Don't do a sub module here. This will just make packaging difficult and risk of getting old versions of the rules.

I opt for skipping this alltogether, and let the server admin take care of syncing whatever rules they want. We can (in a later PR) add a helper script that syncs various sigma repositories that can be run on the server, either manually or via cron.

WDYT?

Reply:
sure, is it okay to then add the data/sigma folder into .gitignore?
For the time being I will then add sections in one or more docs to explain how to pull the repo and or place the own rules.

Reply:
ok adressed in https://github.com/jaegeral/timesketch/commit/c778e1de1ed53e1127518b296768246603ca8db2